scripts/observers/start.py

```diff
@@ -0,0 +1,41 @@
+import os
+import sys
+from pathlib import Path
+from dotenv import load_dotenv
+
+load_dotenv()
+
+working_dir = str(Path.home()) + str(os.getenv('WORKING_DIRECTORY'))
+observer_dir = str(os.getenv('OBSERVER_DIR'))
+
+
+def start_observer(shard_id):
+    current_observer = observer_dir + str(shard_id)
+    working_dir_observer = working_dir + current_observer
+
+    current_directory = os.getcwd()
+    # start observer
+    os.chdir(working_dir_observer + "/node")
+    command = "./node" + " --log-level *:DEBUG --no-key --log-save"
+    # os.system("gnome-terminal 'bash -c \"" + command + ";bash\"'")
+    os.system("screen -d -m -S obs" + str(shard_id) + " " + command)
```

Comment:
Oh, I see, we're using `screen`. Perhaps get a second opinion from @valentin-lup, if this is all right.